yang-module-versioning/draft-ietf-netmod-yang-module-versioning.xml

```diff
@@ -447,17 +451,16 @@ revision 2019-01-02 {
     </figure>
   </section>
   
-  <section anchor="node-tags" title="Schema node compatibility extension statements">
-    <t>In addition to the revision extension statement in <xref target="revision_nbc_marker"/>, YANG extension statements are also available to indicate compatibility information
-    for certain individual YANG statements.
+  <section anchor="node-tags" title="Node compatibility extension statements">
+    <t>In addition to the revision extension statement in <xref target="revision_nbc_marker"/>, YANG extension statements are also available to indicate compatibility information for schema nodes and certain individual YANG statements.
     </t>
```

Comment:
Add a paragraph to say schema node as per 7950 and the statements listed in the YANG module defiition of nbc-change-at (forward ref).

Reply:
done in local draft - will be part of next checkin in this PR.